app/src/main/java/com/onboarding/calculator/mvp/view/CalculatorView.java

```diff
@@ -19,4 +23,18 @@ public CalculatorView(Activity activity, ActivityMainBinding binding) {
     public void showValues(String value) {
         binding.textViewResult.setText(value);
     }
+
+    @Override
+    public void showError() {
+        binding.textViewResult.setText(R.string.calculator_error);
+    }
+
+    @Override
+    public void showDeleteAllMessage(){
+        Toast.makeText(getContext(),MESSAGE_DELETE_ALL,Toast.LENGTH_LONG).show();
+    }
+    @Override
+    public void showDeleteMessage(){
+        Toast.makeText(getContext(),MESSAGE_DELETE_VALUE,Toast.LENGTH_LONG).show();
```

Comment:
showDeleteAllMessage() and showDeleteMessage() are the same...maube you can use a common method with a parameter, to sent the text message toshow

Reply:
Solved!

Reply:
comment not fixed,